src/objectid.ts

```diff
@@ -212,7 +212,7 @@ export class ObjectId {
     }
 
     if (otherId instanceof ObjectId) {
-      return this.toString() === otherId.toString();
+      return this.id[11] === otherId.id[11] && this.id.equals(otherId.id);
```

Comment:
should we be using `this[kId]` instead of `this.id`?

Reply:
I think it would be cleaner... i mean for what do we have the symbol

Reply:
```suggestion
      return this.id[11] === otherId.id[11] && this[kId].equals(otherId[kId]);
```

Reply:
lets see if the tests run through

Reply:
We should use `kId` in the entire statement

```typescript
return this[kId][11] === otherId[kId][11] && this[kId].equals(otherId[kId]);
```

Reply:
done